src/device-state.ts

```diff
@@ -684,24 +684,29 @@ export function reportCurrentState(newState: DeviceReport = {}) {
 	emitAsync('change', undefined);
 }
 
-export async function reboot(force?: boolean, skipLock?: boolean) {
-	await updateLock.abortIfHUPInProgress({ force });
-	await applicationManager.stopAll({ force, skipLock });
-	logger.logSystemMessage('Rebooting', {}, 'Reboot');
-	const $reboot = await dbus.reboot();
-	shuttingDown = true;
-	emitAsync('shutdown', undefined);
-	return await $reboot;
+export async function reboot(force: boolean) {
+	// Multi-app warning!
```

Comment:
To clarify, we'll need to change this when multiple user apps are possible, because updateLock.lock in this case needs to check that none of the multiple apps have locks taken, but right now the lock method only accepts 1 appId.